### ISSUE TITLE
MB-BOLD timepoints limitation

### ISSUE THREAD

**Issue by rwmair** [CMRR-C2P/MB#263]
I'm helping a group that acquires MB-BOLD scans with very short TR's, and wants to run very long scans, so the inherent limit of 4096 BOLD timepoints is a problem. 

Is there any chance this could be increased in R017? (eg 16k or 32k) Its hopefully just a redefinition of the upper limit for this parameter - unless very large numbers of BOLD timepoints cause some other downstream problem.

Thanks

**Comment by eauerbach** [comment]
Yes, this is in R017, the new limit is 32k. Reminder that it needs more testing. IIRC some of the postprocessing tools like GLM will crash if it sees more than 4096 images, this issue previously came up with multi-echo (#209).

**Comment by rwmair** [comment]
Awesome - great news. Thanks. We'll be glad to test it out when R017 is released, tho incompatibility with GLM post-processing is not an issue for us.
